### PR TITLE
refactor: update `blas/ext/base/dapxsumpw` to follow current project conventions

### LINKED ISSUE #1469: [RFC]: refactor `blas/ext/base/dapxsumpw` to follow current project conventions
### Description

This RFC proposes refactoring [`@stdlib/blas/ext/base/dapxsumpw`](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/blas/ext/base/dapxsumpw) to follow current project conventions.

These conventions are outlined in more detail in https://github.com/stdlib-js/stdlib/issues/788. Namely, we want to migrate from C++ add-on interfaces to C add-on interfaces and then make various style and simplification changes.

Some example commits which perform most, if not all, of the requested changes include https://github.com/stdlib-js/stdlib/commit/d66d80cfcbab1da95cb5ae8cfae4a1fc4c5129d1 (saxpy),  https://github.com/stdlib-js/stdlib/commit/3c82fe7b6c0f9fd6ef02c86d6e2024d708ac8c14 (cswap), and https://github.com/stdlib-js/stdlib/commit/5b15c9675be434c0e11b1ec1e1f78d7b089abbcd (dasum).

### Related Issues

Related issues #788, #1152.

### Questions

No.

### Other

No.

### Checklist

- [X] I have read and understood the [Code of Conduct](https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md).
- [X] Searched for existing issues and pull requests.
- [X] The issue name begins with `RFC:`.

### PR DESCRIPTION
**Pull request by ShivamAhir**
Resolves # .

## Description

This pull request:

-   a
-   b
-   c

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves #1469 
-   fixes #

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [ ] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/blas/ext/base/dapxsumpw/README.md [hunk 1]
```diff
@@ -138,18 +131,11 @@ var v = dapxsumpw.ndarray( N, 5.0, x, 2, 1 );
...
 var dapxsumpw = require( '@stdlib/blas/ext/base/dapxsumpw' );
 
-var x;
-var i;
-
-x = new Float64Array( 10 );
-for ( i = 0; i < x.length; i++ ) {
-    x[ i ] = round( randu()*100.0 );
-}
+var x = filledarrayBy(10, 'float64', discreteUniform(0, 100));
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/blas/ext/base/dapxsumpw/README.md]:
```suggestion
var x = filledarrayBy( 10, 'float64', discreteUniform( 0, 100 ) );
```

File: lib/node_modules/@stdlib/blas/ext/base/dapxsumpw/manifest.json [hunk 2]
```diff
@@ -34,7 +34,12 @@
 				"-lm"
 			],
 			"libpath": [],
-			"dependencies": []
+			"dependencies": [
+				"stdlib/napi/export.h",
+				"stdlib/napi/argv.h",
+				"stdlib/napi/argv_int64.h",
+      			"stdlib/napi/argv_strided_float64array.h"
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/blas/ext/base/dapxsumpw/manifest.json]:
```suggestion
				"@stdlib/napi/export",
				"@stdlib/napi/argv",
				"@stdlib/napi/argv-int64",
      			"@stdlib/napi/argv-strided-float64array"
```

File: lib/node_modules/@stdlib/blas/ext/base/dapxsumpw/manifest.json [hunk 3]
```diff
@@ -34,7 +34,12 @@
 				"-lm"
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/blas/ext/base/dapxsumpw/manifest.json]: Indentation should be changed to two spaces here.

Would you please also update the file to have separate configurations for benchmarks and examples. The linked changes in the RFC or merged PRs should give guidance on how the file has to be updated.

**Review by Planeshifter — CHANGES_REQUESTED** [review]
Almost ready for merge; just needs updates to the `manifest.json` I believe. Thanks for your efforts!

**Comment by ShivamAhir** [comment]
@Planeshifter  I think I understood what you wanted me to correct, and I have made the necessary adjustments, including adding the missing include and updating the dependencies in the ```manifest.json``` file. Thank you😊  for helping me identify and resolve my errors.

**Review by Planeshifter — APPROVED** [review]
Thanks for the changes! 

Had to fix the indentation in the manifest file (make sure you have an `.editorconfig` extension for your code editor installed, then the files should be auto-formatted on save), but otherwise looks all good to me now. Will merge shortly after CI has passed.

**Pull request merged**